src/Nav/Nav.tsx

```diff
@@ -10,6 +10,9 @@ const Nav: FC = () => {
 					<Grid item>
 						<Link href="/">Home</Link>
 					</Grid>
+					<Grid item>
+						<Link href="/SearchUsers">Search Users</Link>
```

Comment:
Same comment about the URL as in the other file

```suggestion
						<Link href="/search-users">Search Users</Link>
```